infra/terraform/testground.tf

```diff
@@ -66,35 +66,44 @@ resource "aws_security_group" "default" {
   }
 }
 
+data "aws_ami" "base" {
+  most_recent = true
+  owners      = ["self"]
+  filter {
+    name   = "tag:Name"
+    values = [var.aws_ami]
+  }
+}
+
 resource "aws_instance" "testground_manager" {
-  ami                    = "${lookup(var.aws_amis, var.aws_region)}"
-  availability_zone      = "${var.aws_availability_zone}"
-  instance_type          = "${var.aws_instance_type_manager}"
-  key_name               = "${var.key_name}"
+  ami                    = data.aws_ami.base.id
```

Comment:
If we are to use these scripts for long-lived infrastructure for Testground, we might want to lock the `ami` to a constant value. Otherwise a few days/weeks after we run `terraform apply` for the first time, Terraform would start suggesting that we replace the VMs with the newest AMI, which becomes a problem if we want to apply other changes to the infrastructure.

Reply:
I previously had the hard-coded AMIs (there's a different AMI for each region), but I think it's going to be more flexible to use AWS Tags and the matching logic (see `data "aws_ami" base {}`) ... that way, we don't don't have to update them every single time we change the Packer image, but we can still version things with the tags).